dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/dto/DatasetDTO.java

```diff
@@ -17,6 +17,7 @@
     private String globalIdCreateTime;
     private String publisher;
     private String publicationDate;
+    private boolean isGuestbookAvailable;
```

Comment:
I'm guessing this is info if guestbook was selected? Since it can be available without being selected and then it shouldn't influence metadata (if I remember this correctly?). If that's the case I would go for "isAnyGuestbookSelected".

Reply:
Since there can be only one guestbook active I'll name it `hasActiveGuestbook`

Reply:
I would also add javadoc above getter to explain why we need this field.